### ISSUE TITLE
Can't Seem to Assign Primary Display Number for Steam Link

### ISSUE THREAD

**Issue by ruineka** [ValveSoftware/steam-for-linux#8282]
I am vigorously testing game streaming on Linux and have noticed that when under a Display Port/HDMI multi monitor setup big picture mode when launched with the Steam Link app opens up on Display 1 (Regardless of what the primary display setting is set on Gnome), I am unable to reassign the numbers due to a hardware restriction so we need a way to point Steam to what display is actually the primary so it knows where big screen mode should be.

Long story short is when I launch Steam Link from my living room big picture mode is not visible, instead you are prompted to the Linux desktop. (HDMI is default Display 1, but DisplayPort is my actual primary monitor which is listed as Display 2) I have to drag big picture mode from one display to the other for it to be visible on the Steam Link device.

**Comment by kisak-valve** [comment]
Hello @ruineka, please use https://steamcommunity.com/app/353380/discussions/10/ for feedback on the Steam Link for Linux client.
